.travis.yml

```diff
@@ -14,17 +14,31 @@ before_install:
 
 install: sudo make install_pip
 
+before_script:
+  - |
+    wget https://github.com/openshift/origin/releases/download/v3.6.0/openshift-origin-client-tools-v3.6.0-c4dd4cf-linux-64bit.tar.gz -O ~/openshift-origin-client-tools-v3.6.0-c4dd4cf-linux-64bit.tar.gz
+    tar -xzvf ~/openshift-origin-client-tools-v3.6.0-c4dd4cf-linux-64bit.tar.gz
+    sudo ln -s $PWD/openshift-origin-client-tools-v3.6.0-c4dd4cf-linux-64bit/oc /usr/bin/oc
+    tmp=`mktemp`
+    echo 'DOCKER_OPTS="$DOCKER_OPTS --insecure-registry 172.30.0.0/16"' > ${tmp}
+    sudo mv ${tmp} /etc/default/docker
+    sudo mount --make-shared /
+    sudo service docker restart
+    docker info
+    export PATH=$PATH:$PWD/openshift-origin-client-tools-v3.6.0-c4dd4cf-linux-64bit/
+
```

Comment:
why this setup? ``mtf-env-set`` is not able to do these steps, or this is caused by version of ``docker`` on travis?

Reply:
OpenShift is not on PyPi. I have to download it, untar it, make some steps. I guess, this is not part for `mtf-env-set`.

Reply:
yep, make sense thanks for explanation